src/client/components/pages/entities/cb-review.js

```diff
@@ -0,0 +1,70 @@
+import * as bootstrap from 'react-bootstrap';
+const { Col, Row } = bootstrap;
+import React from 'react';
+import { Rating } from "react-simple-star-rating";
+
+
+function ReviewCard({ reviewData }) {
+    const publishedDate = new Date(reviewData.published_on).toDateString();
+    let reviewText = reviewData.text;
+    if (reviewText.length > 75) {
+        reviewText = reviewText.substring(0, 75) + '...';
```

Comment:
Let's define this number as a constant REVIEW_CONTENT_PREVIEW_LENGTH = 75 and use it below 

```suggestion
    if (reviewText.length > REVIEW_CONTENT_PREVIEW_LENGTH) {
        reviewText = reviewText.substring(0, REVIEW_CONTENT_PREVIEW_LENGTH) + '...';
```